packages/data-broker/src/types/results.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+export interface Analytic {
+  dataElement: string;
+  organisationUnit: string;
+  period: string;
```

Comment:
Minor: comment here would be nice for period property, should be in format YYYYMMDD e.g. "20200103"

Reply:
Good point, I'll add in the next PR :-)